### DISCUSSION TITLE
How do I get_posts from only year 2020

### DISCUSSION THREAD

**Discussion by yolandeyj** [kevinzg/facebook-scraper#866] [General]
Can I use get_posts or get_posts_by_search ?
Can someone give me the command?
Thanks a lot.

**Comment by yolandeyj** [comment]
Can anybody help? Thanks.

**Reply by neon-ninja** [reply]
There's no way, you'll have to extract all posts and filter by year yourself

**Reply by rodriguesprobr** [reply]
Maybe it's important to make some sort of "continue from where it stopped" function.
